qemu/tests/numa_prealloc_threads.py

```diff
@@ -2,12 +2,22 @@
 
 from avocado.utils import cpu
 from virttest import env_process
-from virttest import error_context
 from virttest.qemu_monitor import QMPCmdError
 from avocado.utils import process
 
 
-@error_context.context_aware
```

Comment:
Why delete it?

Reply:
@zhenyzha 
I replaced the existing log traces (also included new ones) to use directly test.log, I am not a big fan of using error_context but if you consider it necessary or a better idea, I'll change back log traces

Reply:
Don't worry I don't have a preference, just curious.

Reply:
As the explanation [with patch https://github.com/autotest/tp-qemu/pull/3841](https://github.com/autotest/tp-qemu/pull/3841#discussion_r1378412583), `error_context` is useful.
But the whole file does not use `error_context.context` now,  better use error_context for the key steps.

Reply:
Updated, thanks!